SimG4Core/Application/src/StackingAction.cc

```diff
@@ -260,7 +260,8 @@ G4ClassificationOfNewTrack StackingAction::ClassifyNewTrack(const G4Track * aTra
 	  ((pdg == 2112) && (ke < kmaxNeutron))) { classification = fKill; }
     }
     if (!trackNeutrino  && classification != fKill) {
-      if (pdg == 12 || pdg == 14 || pdg == 16 || pdg == 18) 
+      if (pdg == 12 || pdg == 14 || pdg == 16 || pdg == 18 ||
```

Comment:
Right - I had in mind apdg=abs(pdg) as to remove 4 conditionals. Thanks for doing in a near term follow up pr.